shangguanwang/NFT-marketplace
# Create GetStarted View

## Description
The `GetStarted` view should have 3 steps to get started.

## Acceptance Criteria
- [x] Each step should be a single card with image, title, and description.
- [x] Create a separate data.js to store all the data and map it to each `SingleGetStarted` component.

## Related Issue
closes #5 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates
### After
![Get Started View screenshot](https://github.com/shangguanwang/NFT-marketplace/assets/24996005/ea763d9a-dd76-4554-b8c5-2ea65ed16959)

## Testing Steps / QA Criteria
- From your terminal, pull down this branch with `git pull origin feat-getstarted` and check that branch out with `git checkout feat-getstarted`
- Run `npm run dev` to launch the app
- You should see the "Get Started" view
